api/src/operations/read/index.ts

```diff
@@ -0,0 +1,54 @@
+import { Accountability, PrimaryKey } from '@directus/shared/types';
+import { defineOperationApi, toArray } from '@directus/shared/utils';
+import { ItemsService } from '../../services';
+import { Item } from '../../types';
+import { getAccountabilityForRole } from '../../utils/get-accountability-for-role';
+import { parseJSON } from '../../utils/parse-json';
+
+type Options = {
+	mode: 'one' | 'many' | 'query';
+	collection: string;
+	key: PrimaryKey | PrimaryKey[] | null;
+	query: string;
+	permissions: string; // $public, $trigger, $full, or UUID of a role
+};
+
+export default defineOperationApi<Options>({
+	id: 'read',
```

Comment:
Thoughts on renaming the CRUD operations to `read-item` etc.? This would reduce confusion in case we would ever implement `read-file` etc. operations.

Reply:
That's a very good point. Thoughts on reversing the name into `item-read` (and `file-read`) so that `item-read`, `item-create`, `item-delete` etc can "grouped" when sorting alphabetically?

Reply:
I like it 👍